applications/hindsight_feed/feeders/browser_history_summary/browser_history_summary.py

```diff
@@ -26,6 +26,8 @@ def llm_generate(pipeline, prompt, max_tokens):
 else:
     from transformers import AutoTokenizer, AutoModelForCausalLM
 
+    LLM_MODEL_NAME = "meta-llama/Llama-3.2-1B-Instruct"
```

Comment:
Pull from hindsight_server config now

Reply:
what's a good way to do it? I'm getting an error for importing from beyond toplevel package

Reply:
I merged in a fix. I will get with of the manual python path additions soon!